pkg/kubernetes/container_status.go

```diff
@@ -0,0 +1,43 @@
+package kubernetes
+
+import corev1 "k8s.io/api/core/v1"
+
+type ContainerStatus struct {
```

Comment:
I would prefer this to be helper functions instead of having a struct here as I don't see the need for it.

Instead of
```
c := ContainerStatus{containerStatus}
c.IsReady()
```
do:
```
IsReady(containerStatus)
```